src/test/scala/sigmastate/lang/SigmaTyperTest.scala

```diff
@@ -275,6 +275,7 @@ class SigmaTyperTest extends PropSpec with PropertyChecks with Matchers with Lan
     typecheck(env, "SELF.R1[Int].isDefined") shouldBe SBoolean
     typecheck(env, "SELF.R1[Int].isEmpty") shouldBe SBoolean
     typecheck(env, "SELF.R1[Int].get") shouldBe SInt
+//    typecheck(env, "SELF.getReg[Int](1)") shouldBe SOption.SIntOption
```

Comment:
commented code without explanation

Reply:
made it `// TODO    typecheck(env, "SELF.getReg[Int](1)")` in upcoming PR